tron/kubernetes.py

```diff
@@ -148,35 +148,58 @@ def handle_event(self, event: Event) -> None:
                 self.log.error(f"Event with >1 || 0 containers: {raw_object}")
             else:
                 main_container_statuses = container_statuses[0]
-                main_container_state = main_container_statuses.get("state")
-                if main_container_state is None or main_container_state.get("terminated") is None:
+                main_container_state = main_container_statuses.get("state", {}) or {}
+                main_container_last_state = main_container_statuses.get("lastState", {}) or {}
+                if (main_container_state is None or main_container_state.get("terminated") is None) and (
+                    main_container_last_state is None or main_container_last_state.get("terminated") is None
+                ):
```

Comment:
it might aid readability here if we refactor both sides of the `and` to separate variables

e.g.,
```suggestion
                event_missing_state = main_container_state is None or main_container_state.get("terminated") is None
                event_missing_previous_state =  main_container_last_state is None or main_container_last_state.get("terminated") is None
                if event_missing_state and event_missing_previous_state:
```

that said: is this correct? seems like if one of these is fully populated and the other isn't, we'd continue onwards with execution

Reply:
I originally had it as `or` instead of `and` but it was giving undesirable results. I forget why I changed it..